<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QCheckBox" name="cB_dir">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>dir</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_ena">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ena</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sB_rps">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>129</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000000000000000.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>158</y>
     <width>129</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rotation per second</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>691</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>rps</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="hS_M1_2">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sB_M1_2">
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>691</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Motor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="hS_M1">
      <property name="minimum">
       <number>2000</number>
      </property>
      <property name="maximum">
       <number>65000</number>
      </property>
      <property name="value">
       <number>65000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sB_M1">
      <property name="minimum">
       <double>420.000000000000000</double>
      </property>
      <property name="maximum">
       <double>65000.000000000000000</double>
      </property>
      <property name="value">
       <double>65000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lE_cmd"/>
      </item>
      <item>
       <widget class="QPushButton" name="pB_send">
        <property name="text">
         <string>SEND</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pTE_log"/>
    </item>
    <item>
     <widget class="QPushButton" name="pB_parking">
      <property name="text">
       <string>Parking</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>160</y>
     <width>131</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sB_freq">
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000000000000000000.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
